operations/boot_orchestration/Boot_UANs.md

```diff
@@ -8,34 +8,29 @@ UAN boot images and a BOS session template have been created. See [Create UAN Bo
 
 ## Procedure
 
-The commands in this procedure are to be run on a master or worker NCN, except where noted otherwise.
-
-1. Create a BOS session to boot the UAN nodes.
+1. (`ncn-mw#`) Create a BOS session to boot the UAN nodes.
 
     ```bash
-    cray bos session create --template-uuid uan-sessiontemplate-PRODUCT_VERSION \
+    cray bos v2 session create --template-name uan-sessiontemplate-PRODUCT_VERSION \
```

Comment:
I chose to update the specific examples like this to v2, partially to reduce cluttering the docs with two versions of every example, and partially to encourage the use of v2.  At this point if someone knows so little about BOS that they're referencing the docs for this information, I would rather they learn v2.  Let me know if you disagree with that logic.

Reply:
These documents are reference materials. We are supporting both V1 and V2 concurrently. Therefore, I feel like we should have documentation for both, especially because there is not a simple switch of 'v1' for 'v2'.

Reply:
These commands are already in the endpoint specific sections and the cheat sheet.